spec/ruby/core/module/const_missing_spec.rb

```diff
@@ -24,4 +24,8 @@
     end
   end
 
+  it "is called regardless of visibility" do
+    klass = Class.new { extend ConstantSpecs::ProtectedConstMissing }
```

Comment:
Since there is only one usage of this fixture, I would define the method directly in the `Class.new` block, makes it easier to read.

Reply:
I don't fully understand why, but it needs to be extended from a module. If the method is defined directly in the class, the spec would pass on TruffleRuby master, whereas the spec as written here will not.

Reply:
Then it would be nice to add a short comment about it, or have 2 checks, one for the basic case and one with the `extend`.